tests/vmi_lifecycle_test.go

```diff
@@ -1826,6 +1767,13 @@ func renderPkillAllPod(processName string) *k8sv1.Pod {
 	return tests.RenderPrivilegedPod("vmi-killer", []string{"pkill"}, []string{"-9", processName})
 }
 
+func createVMI(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstance {
```

Comment:
- You do not need to pass the virt client, it is a singletone that you can just call for from within the functions.
- The name is not hinting that it is also blocking/waiting, so consider calling it `createVMIAndWaitStart`.
- At the end, you better return an updated object: `return libwait.WaitForSuccessfulVMIStart(vmi)`.

Reply:
> - You do not need to pass the virt client, it is a singletone that you can just call for from within the functions.

I've seen this done elsewhere in the tests - https://github.com/kubevirt/kubevirt/blob/0b979471e383f9004e7cca01e0bf53b5551e8ef2/tests/vm_test.go#L1943
> - The name is not hinting that it is also blocking/waiting, so consider calling it `createVMIAndWaitStart`.

That is a much better name.
> - At the end, you better return an updated object: `return libwait.WaitForSuccessfulVMIStart(vmi)`.

> You could also reuse `RunVMIAndExpectLaunch`

For some reason I thought that function did more like verify the console. I will try to reuse that rather than creating my own.